data/Makefile

```diff
@@ -87,70 +101,139 @@ endif
 .DELETE_ON_ERROR:
 .SECONDARY:
 
-.PHONY: f1chr4 f1chr2R f1 fishchr25 fish
+.PHONY: f1chr4 f1chr2R f1 fishchr25 fish physical-map scaffolds
 all: f1chr4 f1chr2R f1 fishchr25 fish
 
+$(lr).physlr.physical-map: $(lr).k$k-w$w.n$(n1)-$(n2).c2-$(minimizer_multiplicity).physlr.overlap.m$m.mol.mol2-bcs.backbone.path
+	ln -sf $< $@
+
+$(lr).physlr.physical-map.$(ref).n10.paf.gz: $(lr).k$k-w$w.n$(n1)-$(n2).c2-$(minimizer_multiplicity).physlr.overlap.m$m.mol.mol2-bcs.backbone.map.$(ref).n10.paf.gz.pdf
+	ln -sf $< $@
+
+$(draft).physlr.fa: $(lr).k$k-w$w.n$(n1)-$(n2).c2-$(minimizer_multiplicity).physlr.overlap.m$m.mol.mol2-bcs.backbone.map.$(draft).n10.sort.best.bed.path.fa
+
+scaffolds:
+
+ifndef ref
+scaffolds: \
+	physical-map \
+	$(draft).physlr.fa
+else
+scaffolds: \
+	physical-map \
+	$(draft).physlr.quast.tsv
+endif
+
+physical-map:
+
+ifndef ref
+physical-map: \
+	$(lr).physlr.physical-map
```

Comment:
```suggestion
	$(lr).physlr.physical-map.path
```

Reply:
I don't think the path suffix is necessary. Path isn't a standard suffix to begin with so I think labeling physical path will work too

Reply:
Hmm I disagree. That is a suffix we use for ABySS, and I feel like we need to indicate the file format generally somehow

Reply:
Well I was going to add in the readme that the output is with x suffix and they are in y format